src/components/MessageCard.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Box, GU, LoadingRing, textStyle, useTheme } from '@aragon/ui'
+
+export default function MessageCard({
+  icon,
+  title,
+  paragraph,
+  loading,
+  noBorder,
```

Comment:
What would you think of using `border` with a default value of `true`? I generally try to avoid negative names in params / props.

Reply:
Makes sense!